pkg/expr/mathexp/reduce.go

```diff
@@ -123,7 +123,7 @@ func (s Series) Reduce(refID, rFunc string, mapper ReduceMapper) (Number, error)
 	floatField := Float64Field(*fVec)
 	reduceFunc, err := GetReduceFunc(rFunc)
 	if err != nil {
-		return number, err
+		return number, fmt.Errorf("invalid expression '%s': %w", refID, err)
```

Comment:
I'm not sure if `expression` is the best word to use here, perhaps we could just use Ref ID?

Reply:
that's exactly how they're called in UI.